### PR TITLE
Initialize pancan

### PR DESCRIPTION
**Pull request by ssettipalli**
look at https://broadinstitute.atlassian.net/browse/DDP-6304
Initialize PanCan study
Auth0 and skeleton of the study
Prequal is just placeholder and not as per requirement
Some conf files has samples which can be used during activities implementation.

Helps build/implement other backend tickets and view front-end static pages

### CODE REVIEW

File: study-builder/studies/pancan/study-activities.conf [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    {
+      "filepath": "icons/in_progress.svg",
+      "statusType": "IN_PROGRESS"
+    },
+    {
+      "filepath": "icons/complete.svg",
+      "statusType": "COMPLETE"
+    }
+  ],
+  "activityTimestamp": "2021-07-01T00:00:00Z",
```
Comment (yufengwng) [inline study-builder/studies/pancan/study-activities.conf]: PanCan is a new study, not a migrated one. Let's remove this timestamp.

**Review by MocanaAtBroad — DISMISSED** [review]

File: study-builder/studies/pancan/about-you.conf [hunk 2]
```diff
@@ -0,0 +1,82 @@
+{
+  include required("../snippets/activity-general-form.conf"),
+  "studyGuid": "cmi-pancan",
+  "activityCode": "ABOUTYOU",
```
Comment (yufengwng) [inline study-builder/studies/pancan/about-you.conf]: I recommend pulling out  identifiers into the `subs.conf` file, similar to how we're doing it for `testboston` and `rgp`.

**Review by yufengwng — DISMISSED** [review]

File: study-builder/studies/pancan/substitutions.conf [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (yufengwng) [inline study-builder/studies/pancan/substitutions.conf]: Can we rename this file to `subs.conf`? (I really don't want to type out the whole thing...)

Comment (ssettipalli) [reply study-builder/studies/pancan/substitutions.conf]: I initially had it subs and just before push changed it

File: study-builder/studies/pancan/about-you.conf (hunk 2 above)
Comment (ssettipalli) [reply study-builder/studies/pancan/about-you.conf]: Its just a placeholder for now but will change it

**Review by yufengwng — APPROVED** [review]

**Pull request merged**